x-pack/plugins/cloud_security_posture/server/create_transforms/benchmark_score_transform.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
+import {
+  LATEST_FINDINGS_INDEX_PATTERN,
+  BENCHMARK_SCORE_INDEX_PATTERN,
+} from '../../common/constants';
+
+export const benchmarkScoreTransform: TransformPutTransformRequest = {
+  transform_id: 'cis_kubernetes_benchmark.score-default-0.0.1',
```

Comment:
lets change the id to be `cloud_security_posture` instead of `cis_kubernetes_benchmark`

Reply:
done